Makefile

```diff
@@ -179,12 +180,17 @@ go-version:
 ###                               Go Modules                                ###
 ###############################################################################
 
-go.sum: go.mod verify tidy
+go.sum: go.mod
+	@echo "Ensuring app dependencies have not been modified..."
+	go mod verify
+	go mod tidy
```

Comment:
This reverts a change made when updating the makefile and fixes the failing liveness test. I thought it would be good to run go mod verify and tidy for all modules when running make build and make install but when we do an official release, we should bump all modules and use the module version rather than the replace directive so this should be ok.